examples/mnist/train.py

```diff
@@ -161,17 +135,19 @@ def train_and_evaluate(config: ml_collections.ConfigDict,
 
   for epoch in range(1, config.num_epochs + 1):
     rng, input_rng = jax.random.split(rng)
-    state, train_metrics = train_epoch(
-        state, train_ds, config.batch_size, epoch, input_rng)
-    loss, accuracy = eval_model(state.params, test_ds)
-
-    logging.info('eval epoch: %d, loss: %.4f, accuracy: %.2f',
-                 epoch, loss, accuracy * 100)
-
-    summary_writer.scalar('train_loss', train_metrics['loss'], epoch)
-    summary_writer.scalar('train_accuracy', train_metrics['accuracy'], epoch)
-    summary_writer.scalar('eval_loss', loss, epoch)
-    summary_writer.scalar('eval_accuracy', accuracy, epoch)
+    state, train_loss, train_accu = train_epoch(
+      state, train_ds, config.batch_size, epoch, input_rng)
+    _, test_loss, test_accu = apply_model(
+      state, test_ds['image'], test_ds['label'])
+
+    print(
+      'epoch:% 3d, train_loss: %.4f, train_accu: %.2f, test_loss: %.4f, test_accu: %.2f' %
+      (epoch, train_loss, train_accu * 100, test_loss, test_accu * 100))
+
+    summary_writer.scalar('train_loss', train_loss, epoch)
+    summary_writer.scalar('train_accu', train_accu, epoch)
```

Comment:
Let's keep the original name `train_accuracy`.
(Here and elsewhere)

Reply:
done